suprb/optimizer/rule/es/es.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 
 from suprb.rule import Rule, RuleInit
-from suprb.rule.matching import MatchingFunction, OrderedBound
```

Comment:
Do we need those imports here? I think we only need the default one here and the others are specified in the runs itself